api/python/quilt3/session.py

```diff
@@ -23,33 +26,33 @@
 
 AUTH_PATH = BASE_PATH / 'auth.json'
 CREDENTIALS_PATH = BASE_PATH / 'credentials.json'
-VERSION = pkg_resources.require('quilt3')[0].version
+VERSION = metadata.version('quilt3')
 
 
 def _load_auth():
     if AUTH_PATH.exists():
-        with open(AUTH_PATH, encoding='utf-8') as fd:
+        with AUTH_PATH.open(encoding='utf-8') as fd:
```

Comment:
Is this the preferred way? I can't find anything about which `open` is better.

In any case, if this way is better, could you do it in a separate PR, just cause it's not related to `pkg_resources`?